templates/disk/test_disk.py

```diff
@@ -35,114 +35,197 @@ def setUp(self):
                                     ],
                                 )
 
-    def test_validate(self):
+    def test_validate_success_1(self):
```

Comment:
I really don't like the numbers in the function names. They're not descriptive at all. You can add something in the name that points to what's different about it from the other tests. Something like `test_validate_success_with_name` and `test_validate_success_with_diskid`. Same for the failure tests.